cfgov/jobmanager/jinja2/jobmanager/job_listing_page.html

```diff
@@ -297,19 +251,25 @@ <h2 class="a-heading">
         "jobLocation" : {
             "@type": "Place",
             "address": [
-            {% if location_type == 'office' and page.allow_remote -%}
+            {% if page.allow_remote -%}
                 {
                     "@type": "PostalAddress",
                     "addressCountry": "US"
                 }
-            {% elif cities %}
-                {%- for city in cities -%}
-                    {
-                        "@type": "PostalAddress",
-                        "addressLocality": "{{ city.name }}",
-                        "addressRegion": "{{ city.state.abbreviation }}",
-                        "addressCountry": "US"
-                    }{{ ', ' if not loop.last else '' }}
+			{% else -%}
+				{%- set city_lists =
+					regions | map( attribute='major_cities' ) | list +
+                    [ offices ]
```

Comment:
Is the formatting correct here? Looks jumbled.

Reply:
Thanks, this was a tab/spaces thing. Will fix.